cmd/keytransparency-server/frontend.go

```diff
@@ -193,15 +199,21 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to create SQL history: %v", err)
 	}
-	sths, err := appender.New(context.Background(), sqldb, *mapID, *mapLogURL, nil)
+
+	tlog, err := config.LogClient(*logID, *logURL, *logPubKey)
 	if err != nil {
-		log.Fatalf("Failed to create appender: %v", err)
+		log.Fatalf("LogClient(%v, %v, %v): %v", *logID, *logURL, *logPubKey, err)
+	}
+	admin := admin.NewStatic()
```

Comment:
The name of this variable is a bit confusing, how about `static` instead, which is what `NewStatic` returns?